From 6cf0905f4337c0ae045cbb492c8b69212c1182d0 Mon Sep 17 00:00:00 2001
From: "geoffroy.weisenhorn" <weisenhorn.geoffroy@intel.com>
Date: Mon, 26 Oct 2015 18:20:15 +0100
Subject: [PATCH] Fix Netfilter configuration for enabling bandwidth

Fix regression for ndc bandwidth activation, disable netfilter
extension "owner" to have data usage menu visible again.

Change-Id: I1886c4543cb0fc34091d8e77025b922d8dc89c25
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-6405
Signed-off-by: geoffroy.weisenhorn <weisenhorn.geoffroy@intel.com>
---
 arch/x86/configs/x86_64_defconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: b/arch/x86/configs/x86_64_defconfig
===================================================================
--- a/arch/x86/configs/x86_64_defconfig	2016-03-08 16:43:21.811789773 -0800
+++ b/arch/x86/configs/x86_64_defconfig	2016-03-08 16:44:03.204222564 -0800
@@ -869,7 +869,7 @@
 # CONFIG_NETFILTER_XT_MATCH_MULTIPORT is not set
 # CONFIG_NETFILTER_XT_MATCH_NFACCT is not set
 # CONFIG_NETFILTER_XT_MATCH_OSF is not set
-CONFIG_NETFILTER_XT_MATCH_OWNER=y
+# CONFIG_NETFILTER_XT_MATCH_OWNER is not set
 CONFIG_NETFILTER_XT_MATCH_POLICY=y
 # CONFIG_NETFILTER_XT_MATCH_PKTTYPE is not set
 CONFIG_NETFILTER_XT_MATCH_QTAGUID=y
